projects/aca-folder-rules/src/lib/rules-list/rule/rule.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="aca-rule">
+  <div class="aca-rule-checkbox">
+    <mat-checkbox></mat-checkbox>
+  </div>
```

Comment:
I'd remove the checkbox for now until we're ready to add multi-select. It's not necessary for now.